examples/run_workflows_ReactMACE.py

```diff
@@ -0,0 +1,40 @@
+# Author: Igor Kowalec
+'''
+This example shows how to use the work flow functionality with CARMM to optimise
+reactants/products and find transition states in an automated manner using the MACE-MP force field.
+'''
+
+
+def test_run_workflows_ReactAims():
```

Comment:
Small error - still retains name of old ReactAims.

Reply:
> Small error - still retains name of old ReactAims.

now resolved